frontend/components/Input.tsx

```diff
@@ -1,14 +1,19 @@
 import axios from "axios";
 import { FormEvent, useState } from "react";
+import { ITask } from "../src/App";
 
-function Input(props: any): JSX.Element {
+function Input(props: {
+  submited: boolean;
+  setSubmited: (Submit: boolean) => void;
+  list: ITask[];
+  setList: (list: ITask[]) => void;
+}) {
```

Comment:
The prop type definition seems like it is being reused; for example, in the `Task` component. You could extract this into an `interface` and pass it to the necessary components to remove this duplication. This is a suggestion.